kernel/build-kernel.sh

```diff
@@ -0,0 +1,252 @@
+#!
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+#---------------------------------------------------------------------
+# Description: This script is the *ONLY* to build a kernel for development.
+# Note: The default kernel version will be taked from:
+# https://github.com/kata-containers/runtime/blob/master/versions.yaml
+#---------------------------------------------------------------------
+
+# Display usage to stdout.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly script_name="$(basename "${BASH_SOURCE[0]}")"
+readonly script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+#flags and default values
+# GOPATH used by Go tools.
+GOPATH=${GOPATH:-${HOME}/go}
```

Comment:
Hi~It may fail when GOPATH is a set of directories.[#86](https://github.com/kata-containers/osbuilder/pull/86)
maybe `${GOPATH%%:*}` to acquire the first value.😁